internal/test/wmcb/wmcb_test.go

```diff
@@ -88,9 +84,29 @@ type pkgInfo struct {
 	shaType string
 }
 
+func initializeCniPluginsPkgInfo() error {
+	cniPluginsURL, err := framework.GetLatestCniPluginsURL()
```

Comment:
I think, we need to get the package from the called function in framework instead of populating `pkg` structure here or atleast some method that takes the inputs from the called function and populates the pkg structure.

Reply:
@ravisantoshgudimetla I do not understand how the current method is different from what you are suggesting. The method initializeCniPluginsPkgInfo() takes inputs from framework.GetLatestCniPluginsURL() and framework.GetLatestCniPluginsSHA() functions and populates the pkg structure

Reply:
> This function will not return any errors and might set nil value for latestPluginURL, maybe call a function that can return error?
> Discussed this and would like another opinion @suhanime ?

latestCNIPluginURL value will never be nil so I don' really think we need a separate function to return error. @ravisantoshgudimetla any thoughts on this?

Reply:
> @ravisantoshgudimetla I do not understand how the current method is different from what you are suggesting. The method initializeCniPluginsPkgInfo() takes inputs from framework.GetLatestCniPluginsURL() and framework.GetLatestCniPluginsSHA() functions and populates the pkg structure

@ravisantoshgudimetla Addressed this as per our discussions